Makefile

```diff
@@ -336,25 +336,15 @@ run-capella/readonly: capella/readonly
 		-e GIT_ENTRYPOINT=$(GIT_REPO_ENTRYPOINT) \
 		-e GIT_REVISION=$(GIT_REPO_BRANCH) \
 		-e GIT_DEPTH=$(GIT_REPO_DEPTH) \
-		-e GIT_USERNAME="" \
-		-e GIT_PASSWORD="" \
+		-e GIT_USERNAME="$(GIT_USERNAME)" \
+		-e GIT_PASSWORD="$(GIT_PASSWORD)" \
 		$(DOCKER_PREFIX)capella/readonly:$$(echo "$(DOCKER_TAG_SCHEMA)" | envsubst)
 
-run-capella/readonly-debug: capella/readonly
+run-capella/readonly-json: capella/readonly
 	docker run $(DOCKER_RUN_FLAGS) \
-		-it \
-		-v /tmp/.X11-unix:/tmp/.X11-unix \
-		-v $$(pwd)/local/scripts:/opt/scripts/debug \
-		-e DISPLAY=:0 \
 		-p $(RDP_PORT):3389 \
 		-e RMT_PASSWORD=$(RMT_PASSWORD) \
-		-e GIT_URL=$(GIT_REPO_URL) \
-		-e GIT_ENTRYPOINT=$(GIT_REPO_ENTRYPOINT) \
-		-e GIT_REVISION=$(GIT_REPO_BRANCH) \
-		-e GIT_DEPTH=$(GIT_REPO_DEPTH) \
-		-e GIT_USERNAME="$(GIT_USERNAME)" \
-		-e GIT_PASSWORD="$(GIT_PASSWORD)" \
-		--entrypoint bash \
+		-e GIT_REPOS_JSON="$$(cat local/readonly.json)" \
```

Comment:
Shouldn't we make the path here configurable? So use something like GIT_REPOS_JSON_PATH with the default value being `local/readonly.json`?